draft-birkholz-scitt-architecture.md

```diff
@@ -454,6 +442,24 @@ Once the Statement is serialized with the correct content type, the Issuer shoul
 
 Once all the Envelope headers are set, the Issuer MAY use a standard COSE implementation to produce the serialized Claim (the SCITT tag of `COSE_Sign1_Tagged` is outside the scope of COSE, and used to indicate that a signed object is a Claim).
 
+## Standard registration policies
+
+> **Editor's note**
+>
+> The technical design for signalling and verifying registration policies is a work in progress.
+> We expect that once the formats and semantics of the registration policy headers are finalized, standardized policies may be moved to a separate draft.
+> For now, we inline some significant policies to illustrate the most common use cases.
+
+All TS implementations MUST support the following registration policies, and MUST check that all the policies defined in the `reg_info` map are satisfied before a claim can be registered. If the `reg_info` map of a claim contains an unknown policy entry, the claim MUST be rejected. This is to ensure that all verifiers get the same guarantee out of the registration policies regardless of where it is registered.
```

Comment:
This is too strong. We can't expect all implementations to support all listed policies.